src/data/immutable_data.rs

```diff
@@ -87,12 +87,14 @@ impl Debug for ImmutableData {
 #[cfg(test)]
 mod tests {
     use super::*;
-    use hex::ToHex;
     use maidsafe_utilities::{SeededRng, serialisation};
     use rand::Rng;
 
     #[test]
+    #[cfg(not(feature = "use-mock-crypto"))]
```

Comment:
That test should work with or without mock crypto, shouldn't it? It's just the value of `expected_name` below that needs to be conditionally compiled.